init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE users (
```

Comment:
would be better create table if it exist. it will deliver us from additional errors handling. Because we need to check error to be sure that error because of table exist already exist, also error can be if database not available etc.